src/things.ts

```diff
@@ -50,22 +50,27 @@ export function buildTasksFromSQLRecords(
   const tasks: Record<string, ITask> = {};
   taskRecords.forEach(({ tag, ...task }) => {
     const id = task.uuid;
+    if(!id) { return } // Sometimes a record with no data is present
```

Comment:
I think it makes sense to filter out empty records inside `getTasksFromThingsLogbook` instead.

Reply:
I dug a bit more to figure _why_ there were empty records. Turns out that the CSV dump from SQLite had an empty line at the ending of it. A `null` record would be present when `Papa` parsed the CSV.

To fix this I added the `skipEmptyLines` option in the `Papa` config. This means we don't have to have the defensive check here. I also modified the `WHERE` on the checklist query to exclude rows with an empty title.

Overall, I'm quite pleased with how this turned out now.

Reply:
Aha! Really great catch. The null record issue really had me for a loop and I never took the time to debug. Thanks for digging into this further 🙏